draft-ietf-ppm-dap.md

```diff
@@ -2259,6 +2265,7 @@ The Helper handles the aggregation job initialization synchronously:
 PUT /helper/tasks/8BY0RzZMzxvA46_8ymhzycOB9krN-QIGYvg_RsByGec/aggregation_jobs/lc7aUeGpdSNosNlh-UZhKA
 Host: example.com
 Content-Type: application/dap-aggregation-job-init-req
+Content-Length: 100
```

Comment:
I find it odd that the content length doesn't match the length of the example payload.

Reply:
Well, we can't compute a real content length when some of the fields are `agg_param = [0x01, 0x02, ...]`. I think this is good enough for an example. The point is to illustrate that there would be a `Content-Length`.